scylladb/sphinx-scylladb-theme
# fix highlight right sidebar

## Motivation

Since 1.7.1, the right-side navigation highlighting was not behaving as expected. When a heading is clicked, sometimes it incorrectly highlights the previous section instead of the current one.

**Example:** When clicking https://sphinx-theme.scylladb.com/stable/deployment/production.html#setting-up-a-custom-domain, "Installation" is highlighted instead of "Setting up a custom domain".

## How to test

1. Clone the PR.
2. Build the docs.
3. Open http://127.0.0.1:5500/deployment/production/#setting-up-a-custom-domain
4. "Setting up a custom domain" should be highlighted, not "Installation".
